run_container.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-# Build the Docker image
-docker build -t desktop-business-app .
-
-# Run the container in the background (detached mode)
-docker run -d --name desktop-business-app -v $(pwd):/application -w /application desktop-business-app tail -f /dev/null
+docker start desktop-business-app
```

Comment:
I think it is not necessary. When you use create_container which you have introduced, container will be in detached mode. So it's running in the background. It's not required to start it again.